src/main/java/htsjdk/samtools/cram/structure/Container.java

```diff
@@ -116,12 +115,36 @@ public void finalizeContainerState(final Slice... containerSlices) {
         }
     }
 
+    /**
+     * Retrieve the list of CRAI Index entries corresponding to this Container.
+     *
+     * @return the list of CRAI Index entries
+     */
     public List<CRAIEntry> getCRAIEntries() {
         return Arrays.stream(slices)
                 .map(Slice::getCRAIEntry)
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Retrieve the list of CRAI Index entries corresponding to this Container.
+     *
+     * TODO: investigate why we sometimes split multi-ref Slices
```

Comment:
i.e. why do we need both this method and the one above it?

this replaces `CRAIIndex.processContainer()`